frontend/package.json

```diff
@@ -16,7 +16,7 @@
     "reactstrap": "^7.1.0"
   },
   "devDependencies": {
-    "eslint": "5.12.0",
+    "eslint": "^5.12.0",
```

Comment:
We want eslint to be pinned to this specific version, so we don't want the `^`. Which means we also need to reset the package-lock changes.